.github/workflows/ci_release.yml

```
@@ -0,0 +1,69 @@
+name: Release
+on:
+  push:
+  schedule:
+    # Run daily at 1:15am
+    - cron: "15 1 * * *"
+  workflow_dispatch:
+    # Inputs the workflow accepts.
+    inputs:
+      version:
+        # Friendly description to be shown in the UI instead of 'name'
+        description: "Semver type of new version (major / minor / patch)"
+        # Input has to be provided for the workflow to run
+        required: true
+        type: choice
+        options:
+          - patch
+          - minor
+          - major
+
+jobs:
+  # Run the linting and tests
+  hadolint:
+    uses: SkynetLabs/.github/.github/workflows/reusable_dockerfile_lint.yml@master
+
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "1.18"
+      - name: Install analyze
+        run: go install gitlab.com/NebulousLabs/analyze@latest
+      - name: Install golangci-lint
+        run: curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.45.0
+      - name: Lint
+        run: make lint
+      - name: Run unit tests
+        run: make test
+
+  # Check if there were any changes since the last tag if this is not a push
+  # event
+  changes:
+    needs: [hadolint, test]
+    runs-on: ubuntu-latest
+    outputs:
+      updates: ${{steps.changes.outputs.any == 'true'}}
+    if: ${{ github.event_name != 'push' }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # Required due to the weg Git works, without it this action won't be able to find any or the correct tags
```

```suggestion
          fetch-depth: 0 # Required due to the way Git works, without it this action won't be able to find any or the correct tags
```

Should be this I think.